src/NBitcoin/NetworkOptions.cs

```diff
@@ -1,27 +1,41 @@
-﻿namespace NBitcoin
+﻿using System;
+
+namespace NBitcoin
 {
     /// <summary>
     /// The current use-case for this class is to provide option-dependent serialization.
     /// It is a drop-in replacement for the legacy TransactionOptions enumeration.
-    /// 
+    /// </summary>
+    /// <description>
     /// The NetworkOptions instance is intended to be relayed from the Network object to 
     /// the child objects: Network => Block => Transaction => TxIn/TxOut => Coin => ...
     /// It can also be used on repository or stream objects that need to deserialize the 
     /// above objects or passed as a parameter futher down the call-hierarchy as needed.
-    /// </summary>
+    /// </description>
     public class NetworkOptions
     {
         public const uint None = 0x00000000;
+        public const uint POS = 0x20000000;
```

Comment:
NetworkOptions can be cloned and undergo changes as it is passed from the Network to the Block, Transaction and further down the hierarchy for during serialization. If you look at issue 676 I have an example of how NetworkOptions would be changing. At the level where the Block creates a (default) transaction we can look at the Network type (the type that the NetworkOptions are derived from) and create the appropriate transaction type. The constructor of the transaction type would set the POS flag within the NetworkOptions that are passed on from that point. I came to this solution by concluding that we should not really try to say to much about a block chain at the network level.

Reply:
>The constructor of the transaction type would set the POS flag within the NetworkOptions

Perhaps but you can still use POS as its own flag instead of part of the network serialization Flag